src/ReactEditableSvgLabel.js

```diff
@@ -52,10 +52,16 @@ var ReactEditableSvgLabel = React.createClass({
   },
 
   render () {
-    var label = <text ref='label' {...this.props}>{this.props.children}</text>;
+    // Omit onChange, minLabelWidth, and children.
+    var passThroughProps = Object.assign({}, this.props);
+    Object.keys(this.constructor.propTypes).forEach(function (key) {
+      delete passThroughProps[key];
+    });
```

Comment:
This change avoids a warning `Unknown prop `minLabelWidth` on <text> tag.` The idea behind this change is that the component should pass through additional props to the `text` element, but not its own props.